src/checkin/services/flow/step/ValidateVisaStep.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { MockContext } from '@testMocks/model/Context.mock'
+import { Context } from '../../../model/Context'
+import ValidateVisaStep from './ValidateVisaStep'
+
+describe('[ Step / ValidateVisaStep ]', () => {
+  const step = new ValidateVisaStep()
+
+  describe('Test when it should be execute', () => {
+    it('should not execute when they are not flying to the US', () => {
+      const context: Context = new MockContext()
+      context.withSessionBuilder(sessionBuilder => sessionBuilder
+        .data({
+          ...context.getSession().data,
+          country: 'MX',
```

Comment:
I noticed that you wrote the country codes "MX" and "US" many times. Shouldn't it be better to define a constant instead of hardcoding them?